apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -212,18 +215,33 @@ const ApplicationDialogView: React.FC<Props> = ({
     }
 
     try {
-      const object: Record<string, string> = JSON.parse(data);
+      const object = JSON.parse(data);
+
+      if (typeof object === 'object' && object !== null) {
+        for (const [key, value] of Object.entries(object)) {
+          if (!key.trim()) {
+            return t('Keys should not be empty');
+          }
+
+          if (['boolean', 'number'].includes(typeof value) || value === null) {
+            continue;
+          }
```

Comment:
```suggestion
        for (const [key, value] of Object.entries(object)) {
          if (!['boolean', 'number'].includes(typeof value) && !value === null) {
            if (!key.trim()) {
              return t('Keys should not be empty');
            }    
          ...
          }
```
lets rewrite it to remove continue word